src/api/v3/feedback/page_feedback.py

```diff
@@ -12,22 +12,21 @@ def page_feedback(data, region, language, comment, emotion, is_technical):
 
 
 def page_feedback_internal(data, region, language, comment, emotion, is_technical):
-    page_slug = data.get("slug")
-    if not page_slug:
+    page_slug = data.get("slug", None)
+    if page_slug is None or not page_slug:
         return JsonResponse({"error": "Page slug is required."}, status=400)
-    try:
-        page = PageTranslation.objects.get(
-            page__region=region, language=language, slug=page_slug
-        ).page
-        PageFeedback.objects.create(
-            page=page,
-            language=language,
-            emotion=emotion,
-            comment=comment,
-            is_technical=is_technical,
-        )
-        return JsonResponse({"success": "Feedback successfully submitted"}, status=201)
-    except PageTranslation.DoesNotExist:
```

Comment:
Is there a difference beteween your and mine solution in terms of behaviour?
Or is it just the case that your solution is cleaner than mine?

Reply:
Yes, its only a matter of style. As Timo wanted it the other way around, I simply changed it :)

Reply:
And is there a reason, that you only changed this get-request?

Reply:
No, just did not come across the others.

Reply:
There was only one other instance I found: `src/api/v3/feedback/event_feedback.py`. This is now updated as well.